GiniVision/Classes/SupportedFormatsViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SupportedTypesViewController.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 10/19/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+final class SupportedFormatsViewController: UITableViewController {
+    
+    let reuseIdentifier = "reuseIdentifier"
+    let rowHeight: CGFloat = 70
+    let sectionHeight: CGFloat = 70
+    var sections: [(title: String, items: [String], itemsImage:UIImage?, itemsImageBackgroundColor: UIColor)] = [
+        ("Folgende Formate werden unterstützt:",
+         ["Computer-erstellte Überweisungsträger und Rechnungen",
+          "Einseitige Bilder im jpg, png oder tif Format",
+          "PDF Dokumente von bis zu 10 Seiten"],
+         UIImage(named: "supportedFormatsIcon", in: Bundle(for: GiniVision.self), compatibleWith: nil),
+         GiniConfiguration.sharedConfiguration.supportedFormatsIconColor),
+        ("Was nicht analysiert wird:",
+         ["Handschrift",
+          "Fotos von Bildschirme"],
+         UIImage(named: "nonSupportedFormatsIcon", in: Bundle(for: GiniVision.self), compatibleWith: nil),
+         GiniConfiguration.sharedConfiguration.nonSupportedFormatsIconColor)
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = "Unterstützte Formate"
+        tableView.register(SupportedFormatsTableViewCell.self, forCellReuseIdentifier: reuseIdentifier)
+        tableView.rowHeight = rowHeight
+        tableView.tableFooterView = UIView()
+        tableView.sectionHeaderHeight = sectionHeight
+        tableView.allowsSelection = false
+        tableView.backgroundColor = Colors.Gini.pearl
+        tableView.alwaysBounceVertical = false
+        
+        if #available(iOS 11.0, *) { // On iOS is .automatic by default.
```

Comment:
Why only on iOS11? Maybe a comment would be helpful?

Reply:
Because on iOS `contentInsetAdjustmentBehavior` is `automatic` by default and the transition to the target view controller looks weird